atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -344,6 +373,26 @@ def get_kojifile_source_urls(self):
 
         return kojifile_sources
 
+    def get_remote_file_urls(self):
+        """Fetch remote source file urls from all builds
+
+        :return: list, dicts with URL pointing to remote source files
+        """
+        sources = []
+
+        source = self._get_remote_file_urls_helper(self.koji_build)
+        sources.extend(source)
+
+        koji_build = self.koji_build
+
+        while 'parent_build_id' in koji_build['extra']['image']:
```

Comment:
Will this result in endless loop, because condition will be always True?

Reply:
base image doesn't have 'parent_build_id' because it is generated from koji image builder

Reply:
got it now